tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -210,9 +209,8 @@ BuildLambdaString(const std::string &expr, const ColumnNames_t &vars, const Colu
 {
    R__ASSERT(vars.size() == varTypes.size());
 
-   TRegexp re("[^a-zA-Z0-9_]?return[^a-zA-Z0-9_]");
-   Ssiz_t matchedLen;
-   const bool hasReturnStmt = re.Index(expr, &matchedLen) != -1;
+   TPRegexp re(R"(\breturn\b)");
```

Comment:
Viola, fixed. :-)
Not sure if grouping is required here, but will match.

Reply:
it's not grouping, it's the raw string syntax :smile: